indra/newview/llviewermenu.cpp

```diff
@@ -9395,16 +9395,19 @@ void initialize_edit_menu()
 
 }
 
+typedef LLUICtrl::LLCommitCallbackInfo cb_info;
+#define COMMIT_ADD(func_name, func) LLUICtrl::SharedCommitCallbackRegistry::currentRegistrar().add(func_name, cb_info(func))
+#define COMMIT_ADD_TRUSTED(func_name, func) LLUICtrl::SharedCommitCallbackRegistry::currentRegistrar().add(func_name, cb_info(func, cb_info::UNTRUSTED_BLOCK))
+
 void initialize_spellcheck_menu()
 {
-    LLUICtrl::CommitCallbackRegistry::Registrar& commit = LLUICtrl::CommitCallbackRegistry::currentRegistrar();
```

Comment:
So at least in this source, we're not using `CommitCallbackRegistry` at all any more, only `SharedCommitCallbackRegistry`. That makes me feel more positive about simply changing the value type of `CommitCallbackRegistry` to `cb_info`, instead of having a whole separate `SharedCommitCallbackRegistry`.